charts/azure/azure-disk-loss/engine.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
```

Comment:
Can we add csv and icon for this experiment also?
refer: https://github.com/avaakash/chaos-charts/blob/azure-disk-loss/charts/azure/azure-instance-terminate/azure-instance-terminate.chartserviceversion.yamlhttps://github.com/avaakash/chaos-charts/blob/azure-disk-loss/charts/azure/azure-instance-terminate/azure-instance-terminate.chartserviceversion.yaml and https://github.com/avaakash/chaos-charts/tree/azure-disk-loss/charts/azure/icons

Reply:
Also add the entry of this experiment at https://github.com/avaakash/chaos-charts/blob/azure-disk-loss/charts/azure/azure.package.yaml and https://github.com/avaakash/chaos-charts/blob/azure-disk-loss/charts/azure/azure.chartserviceversion.yaml#L15